src/assets/locales/resources_en_US.json

```diff
@@ -14,5 +14,17 @@
   "sky_ag_grid_cell_editor_autocomplete_aria_label": {
     "_description": "aria label for the autocomplete cell editor",
     "message": "Editable autocomplete {0} for row {1}"
+  },
+  "readonly_grid_delete_text": {
+    "_description": "Context menu delete text for readonly grid example",
```

Comment:
Do we really need resource strings for visual tests?

Reply:
ah, this was originally to make sure localization still worked for a different approach to the context menu. Since it's just a standard component now this isn't really necessary.